src/components/PanelWrapper.tsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react';
+import { VscClose } from 'react-icons/vsc';
 
-export function PanelWrapper({ title, children }: { title: string; children: React.ReactNode }) {
+export function PanelWrapper({
+  title,
+  close = false,
+  onCloseClick,
+  children,
+}: {
+  title: string;
+  close?: boolean;
```

Comment:
I think this name is a bit misleading. Shouldn't it be something like `closable` instead?

Reply:
ok. Fair point. Will update this prop.